app/crowdSourceData/parajuliminiyan.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Miniyan Parajuli",
+    "email": "parajuliminiyan@gmail.com",
+    "designation": "Software Engineer",
+    "experience": 1,
+    "skills": [
+      "php",
+      "node",
```

Comment:
@parajuliminiyan  please write nodejs and we are good to go

Reply:
@ujwaldhakal Requested changes resolved.